src/Context/SalesChannelContext/PaymentMethod.php

```diff
@@ -20,6 +20,13 @@ public function getName(): string
         return $this->data['name'];
     }
 
+    public function getTechnicalName(): string
+    {
+        \assert(is_string($this->data['technicalName']) || is_null($this->data['technicalName']));
+        // @deprecated - remove null check after shopware v6.7.0.0 release
```

Comment:
I would add a phpdoc `@since Shopware 6.7.0.0`

releasing a sdk version per shopware version makes no sense, as you want to cover with your app a lot of versions